build_node/build_node_builder.py

```diff
@@ -261,7 +261,7 @@ def __upload_artifacts(self, artifacts_dir,
         return artifacts
 
     def __request_task(self):
-        supported_arches = [self.__config.base_arch]
+        supported_arches = ['src', self.__config.base_arch]
```

Comment:
I'd suggest to leave it up to user, because we have s390x nodes and there are too few workers to use them for sources building